python/lsst/ts/observing/utilities/auxtel/latiss/utils.py

```diff
@@ -0,0 +1,26 @@
+__all__ = ["parse_obs_id"]
+from lsst.ts.observatory.control.constants import atcs_constants
+
+
+def parse_obs_id(obs_id):
+    """Split an obsId into its constituent parts
+
+    E.g.
+        parse_obs_id('AT_O_20200219_000212')
+    """
+    source, controller, dayObs, seqNum = obs_id.split("_")
+    dayObs = f"{dayObs[0:4]}-{dayObs[4:6]}-{dayObs[6:8]}"
+    seqNum = int(seqNum)
+
+    return source, controller, dayObs, seqNum
+
+
+def calculate_xy_offsets(target_position, current_position):
```

Comment:
It feels to me that this method could be part of `ATCS` class. We could probably do a bit more work, make it generic and add it to the `BaseTCS` class.